plugins/base/src/main/kotlin/DokkaBaseConfiguration.kt

```diff
@@ -12,6 +12,7 @@ public data class DokkaBaseConfiguration(
     var customStyleSheets: List<File> = defaultCustomStyleSheets,
     var customAssets: List<File> = defaultCustomAssets,
     var separateInheritedMembers: Boolean = separateInheritedMembersDefault,
+    var homepageLink: String? = null,
```

Comment:
> Do we need to keep backward compatibility for DokkaBaseConfiguration?

I think it's worth moving this var to be the last parameter of the constructor, I think it'll help preserve source compatibility (in case someone imports the class in their build scripts and doesn't use named parameters), so it's at least something. 

Binary compatibility - I'm not sure how much sense it makes to try to preserve it here. Almost every Dokka release is breaking binary compatibility in some way, mostly due to data classes everywhere, so I wouldn't say it's expected that out-of-sync versions will work together flawlessly. And the HTML format will be configured differently once the new Gradle plugin is released, so I'd say we can ignore it for now

Reply:
Yeah, agree, will move parameter to be the last (nor sure, why I haven't done it from the beginning :) )